Package.swift

```diff
@@ -252,10 +252,10 @@ let package = Package(
             name: "WinUIInterop",
             dependencies: []
         ),
-        // .target(
-        //     name: "CursesBackend",
-        //     dependencies: ["SwiftCrossUI", "TermKit"]
-        // ),
+         .target(
```

Comment:
Fix indentation (and for the TermKit package above and TermKitBackend product above that)